docs/reference/mapping.asciidoc

```diff
@@ -137,28 +137,25 @@ A mapping could be specified when creating an index, as follows:
 
 [source,js]
 ---------------------------------------
-PUT my_index?include_type_name=true <1>
+PUT my_index <1>
 {
   "mappings": {
-    "_doc": { <2>
-      "properties": { <3>
-        "title":    { "type": "text"  }, <4>
-        "name":     { "type": "text"  }, <4>
-        "age":      { "type": "integer" },  <4>
-        "created":  {
-          "type":   "date", <4>
-          "format": "strict_date_optional_time||epoch_millis"
-        }
+    "properties": { <2>
+      "title":    { "type": "text"  }, <3>
+      "name":     { "type": "text"  }, <3>
+      "age":      { "type": "integer" },  <3>
+      "created":  {
+        "type":   "date", <4>
+        "format": "strict_date_optional_time||epoch_millis"
       }
     }
   }
 }
 ---------------------------------------
 // CONSOLE
 <1> Create an index called `my_index`.
-<2> Add a mapping type called `doc`.
-<3> Specify fields or _properties_.
-<4> Specify the data `type` and mapping for each field.
+<2> Specify fields or _properties_.
+<3> Specify the data `type` and mapping for each field.
```

Comment:
No docs for item 4

Reply:
callout <4> was deleted (actually <2> was deleted and everything else moved up)

Reply:
Corrected the remaining offenting <4> Callout